build-tools/packages/build-cli/README.md

```diff
@@ -48,6 +48,7 @@ USAGE
 * [`flub rename-types`](docs/rename-types.md) - Renames type declaration files from .d.ts to .d.mts.
 * [`flub run`](docs/run.md) - Generate a report from input bundle stats collected through the collect bundleStats command.
 * [`flub typetests`](docs/typetests.md) - Updates configuration for type tests in package.json files. If the previous version changes after running preparation, then npm install must be run before building.
+* [`flub update`](docs/update.md) - Rewrite imports for Fluid Framework APIs to use the correct subpath import (/alpha, /beta. etc.)
```

Comment:
The command is actually `update fluid-imports`. I can update the config so the docs here aren't specific to the new command. I used update since that's what I used in my other "update project config" command (see #19442). But I'm open to a different top-level verb. What I want to avoid, though, is having multiple top-level commands like `flub adjust-fluid-imports` and `flub adjust-project` etc.

Reply:
Yep - update + fluid-import => looks like advance the version. update is just too generic. Structure is good; so, need to figure out what this class of commands is.

Reply:
I like refactor, or codeshift (since that's what the original library I used to do stuff like this was called, jscodeshift).

Reply:
"Modify?"

Reply:
I like "refactor", "codeshift", and "modify". The first two are pretty code specific (which could be fine if that is the class). "modify" is more open.

Reply:
I plan to change to "modify" but in my latest changes I left it as-is so GitHub comments are still anchored to the right place in the file. Once review is mostly done, I'll move it.